src/components/TabCompleteInput/TabCompleteInput.tsx

```diff
@@ -17,23 +17,30 @@
 
 import * as React from "react";
 
-interface TabCompleteInputProperties {
+interface TabCompleteInputProperties extends React.HTMLProps<HTMLInputElement> {
```

Comment:
tbh this whole `TabCompleteInputProperties` type is unnecessary.  Everything defined here is already defined in `React.HTMLProps<HTMLInputElement>`

Reply:
Hah - 😜